packages/framework/presence/api-extractor/api-extractor-lint-alpha.esm.json

```diff
@@ -0,0 +1,12 @@
+{
+	"$schema": "https://developer.microsoft.com/json-schemas/api-extractor/v7/api-extractor.schema.json",
+	"extends": "<projectFolder>/../../../common/build/build-common/api-extractor-lint.entrypoint.json",
+	"mainEntryPointFilePath": "<projectFolder>/lib/alpha.d.ts",
+	"messages": {
+		"extractorMessageReporting": {
+			"ae-wrong-input-file-type": {
+				"logLevel": "none"
+			}
+		}
+	}
```

Comment:
I am suspicious that only the alpha.esm config has this. I would expect all or none.

Reply:
alpha esm was the only config complaining about src/events/events.ts with this error message: "Error: src/events/events.ts:1:1 - (ae-wrong-input-file-type) Incorrect file type; API Extractor expects to analyze compiler outputs with the .d.ts file extension. Troubleshooting tips: https://api-extractor.com/link/dts-error". 

The error would occur when runnning  'api-extractor run --config api-extractor/api-extractor-lint-alpha.esm.json'. Once I added this it stopped failing, but I'm also not sure why only alpha.esm was having this issue.

Reply:
I guess just run the others one at a time to check their output.